tests/integration/ceph_base_file_test.go

```diff
@@ -233,9 +238,13 @@ func fileSystemCSISnapshotTest(helper *clients.TestClient, k8sh *utils.K8sHelper
 	assert.True(s.T(), k8sh.WaitUntilPVCIsDeleted(defaultNamespace, pvcName))
 
 	logger.Infof("delete snapshotclass")
-
-	err = helper.FSClient.DeleteSnapshotClass(snapshotClassName, snapshotDeletePolicy, namespace)
-	require.NoError(s.T(), err)
+	if !testNFS {
+		err = helper.FSClient.DeleteSnapshotClass(snapshotClassName, snapshotDeletePolicy, namespace)
+		require.NoError(s.T(), err)
```

Comment:
require.NoError can be outside of if else check

Reply:
done :+1: